pkg/middleware/jwt_session_test.go

```diff
@@ -397,6 +397,8 @@ Nnc3a3lGVWFCNUMxQnNJcnJMTWxka1dFaHluYmI4Ongtb2F1dGgtYmFzaWM=`
 		expiresFuture := time.Now().Add(time.Duration(5) * time.Minute)
 		verified := true
 		notVerified := false
+		allowUnverified := true
+		notAllowUnverified := false
```

Comment:
This is really convoluted and does not help with the readability of the code. Please remove the variables and just set true or false accordingly in the field declarations below.